packages/Algernon.package/ALGInputField.class/instance/moveCursorToEnd.st

```diff
@@ -1,5 +1,5 @@
 event handling
 moveCursorToEnd
 	self editor clearSelection.
-	self cursorWrapped: self size.
```

Comment:
I think this behavior should live in cursorWrapped as that method is already concerned with cursor special cases.

Reply:
So should we overwrite the function ? Because currently it's a function only defined on TextMorph.

Reply:
I think that might be a good idea. Sounds like we want to extend the wrapping behavior. What do you think?

Reply:
I see your Point. Just moved the logic to an override.